subprojects/docs/src/main/resources/header.html

```diff
@@ -165,50 +164,98 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
-                <ul id="java-projects">
-                    <li><a href="building_java_projects.html">Building Java &amp; JVM projects</a></li>
-                    <li><a href="java_testing.html">Testing Java &amp; JVM projects</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#advanced-techniques" aria-expanded="false" aria-controls="advanced-techniques">Advanced Techniques</a>
+            <ul id="advanced-techniques">
+                <li><a href="https://guides.gradle.org/using-the-worker-api/">Developing Parallel Tasks</a></li>
+                <li><a href="test_kit.html">Testing a Build with TestKit</a></li>
+                <li><a href="ant.html">Using Ant from Gradle</a></li>
+            </ul>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Example Gradle Projects</a>
+                <ul id="sample-gradle-builds">
+                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL Samples</a></li>
+                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL Samples</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#native-projects" aria-expanded="false" aria-controls="native-projects">C++/Swift Projects</a>
-                <ul id="native-projects">
-                    <li><a href="building_cpp_projects.html">Building C++ projects</a></li>
-                    <li><a href="cpp_testing.html">Testing C++ projects</a></li>
-                    <li><a href="building_swift_projects.html">Building Swift projects</a></li>
-                    <li><a href="swift_testing.html">Testing Swift projects</a></li>
+        </ul>
+
+        <h3 id="managing-dependencies">Managing Dependencies</h3>
+        <ul>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#learning-the-basics-dependency-management" aria-expanded="false" aria-controls="learning-the-basics-dependency-management">Learning the Basics</a>
+                <ul id="learning-the-basics-dependency-management">
+                    <li><a href="core_dependency_management.html">What is Dependency Management?</a></li>
+                    <li><a href="dependency_management_terminology.html">Terminology</a></li>
+                    <li><a href="declaring_repositories.html">Declaring Repositories</a></li>
+                    <li><a href="declaring_dependencies.html">Declaring Dependencies</a></li>
+                    <li><a href="library_vs_application.html">Developing Libraries</a></li>
+                    <li><a href="viewing_debugging_dependencies.html">Debugging Dependencies</a></li>
+                    <li><a href="dependency_resolution.html">Dependency Resolution</a></li>
```

Comment:
How Resolution Works?

Reply:
We should stick with the same pattern for titles. I think "Understanding dependency resolution" would be better.

Reply:
As discussed: Understanding Resolution